### ISSUE TITLE
QGis 3 port

### ISSUE THREAD

**Issue by zezzagio** [blazek/spline#5]
With a last commit in 2015, this project seems a little neglected if not abandoned.
Do you plan to port it to QGis 3?

I did a very clumsy attempt to port it myself; it seems to work (almost), but I'm not very confident on the result. It can be found in the forks, anyway.

Ciao.

**Comment by blazek** [comment]
Yes, abandoned, no plan to port to QGIS 3. If you manage to make a release, I can add you as owner to https://plugins.qgis.org/plugins/spline/ and you can upload the release and continue in development.

**Comment by zezzagio** [comment]
I'm not a programmer; in particular I'm not familiar with Python, and still less with git (I realized I did a mess in the fork.)

I somehow got an almost working thing, which (in someway) runs in QGIS 3, but I can't in conscience release it to unaware and innocent people; civilians must be protected.

I'll keep it for my personal use, at least until I'm more confident in its compliance and robustness.

I'd like, anyway, to see a spline tool integrated in the QGIS digitizing process, as it's done in  the Advanced Digitizings Tool, so to be able to alternate approximated spline paths and ordinary ones.

Ciao.
